src/quo2/components/gradient/gradient_cover/view.cljs

```diff
@@ -0,0 +1,18 @@
+(ns quo2.components.gradient.gradient-cover.view
+  (:require [quo2.components.gradient.gradient-cover.style :as style]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo.theme]
+            [react-native.linear-gradient :as linear-gradient]))
```

Comment:
I think this is the ideal PR to update `react-native-linear-gradient` package to latest version (latest is `2.8.0`, currenlty we are using a version from 4 years ago, `2.5.6`), what do you think @smohamedjavid?

Reply:
Yes @briansztamfater, I checked that too. When I started on this issue, I felt like we should upgrade as the current version is outdated. Then I left that out to keep the PR simple. But, yes, we should update the library. I will upgrade the library in this PR. 👍